src/overlays/gamestates/ovl_file_choose/z_file_select.h

```diff
@@ -84,28 +84,33 @@ typedef enum {
     /* 4 */ SM_FADE_OUT_FILE_INFO,
     /* 5 */ SM_MOVE_FILE_TO_SLOT,
     /* 6 */ SM_FADE_OUT,
-    /* 7 */ SM_LOAD_GAME
+    /* 7 */ SM_LOAD_GAME,
 } SelectMode;
 
 typedef enum {
-    /* 0 */ FS_TITLE_SELECT_FILE,   // "Please select a file."
-    /* 1 */ FS_TITLE_OPEN_FILE,     // "Open this file?"
-    /* 2 */ FS_TITLE_COPY_FROM,     // "Copy which file?"
-    /* 3 */ FS_TITLE_COPY_TO,       // "Copy to which file?"
-    /* 4 */ FS_TITLE_COPY_CONFIRM,  // "Are you sure?"
-    /* 5 */ FS_TITLE_COPY_COMPLETE, // "File copied."
-    /* 6 */ FS_TITLE_ERASE_FILE,    // "Erase which file?"
-    /* 7 */ FS_TITLE_ERASE_CONFIRM, // "Are you sure?"
-    /* 8 */ FS_TITLE_ERASE_COMPLETE // "File erased."
+    /* 0 */ FS_TITLE_SELECT_FILE,    // "Please select a file."
+    /* 1 */ FS_TITLE_OPEN_FILE,      // "Open this file?"
+    /* 2 */ FS_TITLE_COPY_FROM,      // "Copy which file?"
+    /* 3 */ FS_TITLE_COPY_TO,        // "Copy to which file?"
+    /* 4 */ FS_TITLE_COPY_CONFIRM,   // "Are you sure?"
+    /* 5 */ FS_TITLE_COPY_COMPLETE,  // "File copied."
+    /* 6 */ FS_TITLE_ERASE_FILE,     // "Erase which file?"
+    /* 7 */ FS_TITLE_ERASE_CONFIRM,  // "Are you sure?"
+    /* 8 */ FS_TITLE_ERASE_COMPLETE, // "File erased."
```

Comment:
We prefer to not have a comma after the last enum value since it's not technically allowed in C89 (applies to all the others in this file).

Reply:
Oh I mixed this up in my mind. I know OoT generates warnings with trailing enum comma's, but for some reason I thought the warning was if the trailing comma was missing, I mixed it up in my mind. I'll fix it